### ISSUE TITLE
reload_certificates_test: Missing: ['^.*cql_server.*Reloaded.*ccm_node\\.pem\\.*']

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#6507]
Dtest failed in https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/490/testReport/native_transport_ssl_test/NativeTransportSSL/reload_certificates_test/
scylla version 076c8317c78c4cd0cf1dbe70e3da206a14d75326
scylladb/scylla-dtest@df06c04d7dce4f8ac3240866635f5c80b184158f
```
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib64/python3.7/unittest/case.py", line 645, in run
    testMethod()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/native_transport_ssl_test.py", line 168, in reload_certificates_test
    wait_for_cert_reload(node1, "cql_server", ["ccm_node.pem", "ccm_node.key"], from_mark=mark)
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/native_transport_ssl_test.py", line 17, in wait_for_cert_reload
    node.watch_log_for("^.*{}.*Reloaded.*{}\.*".format(module, f.replace('.', '\.')), from_mark=from_mark)
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-ccm/ccmlib/node.py", line 448, in watch_log_for
    [e.pattern for e in tofind]) + ":\n" + reads[:50] + ".....\nSee {} for remainder".format(filename))
ccmlib.node.TimeoutError: 25 May 2020 04:18:25 [node1] Missing: ['^.*cql_server.*Reloaded.*ccm_node\\.pem\\.*']:
```

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/493/artifact/logs-release.2/1590380305393_native_transport_ssl_test.NativeTransportSSL.reload_certificates_test/node1.log:
```
INFO  2020-05-25 04:07:15,599 [shard 0] init - Scylla version 666.development-0.20200524.076c8317c initialization completed.
INFO  2020-05-25 04:07:17,620 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
...
INFO  2020-05-25 04:08:17,676 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
WARN  2020-05-25 04:08:17,679 [shard 1] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key}: std::_Nested_exception<std::runtime_error> (Could not read reloading /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key): std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: open failed: No such file or directory [/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key])
WARN  2020-05-25 04:08:17,679 [shard 0] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key}: std::_Nested_exception<std::runtime_error> (Could not read reloading /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key): std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: open failed: No such file or directory [/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key])
WARN  2020-05-25 04:08:17,683 [shard 0] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.pem, /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key}: std::_Nested_exception<std::runtime_error> (Could not read reloading /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.pem): std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: open failed: No such file or directory [/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.pem])
WARN  2020-05-25 04:08:17,863 [shard 1] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.pem, /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-mx7xvyfj/ccm_node.key}: std::system_error (error GnuTLS:-34, Base64 decoding error.)
```

**Comment by bhalevy** [comment]
Note that this test failed once more recently and the failure looks slightly different.

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/490/testReport/native_transport_ssl_test/NativeTransportSSL/reload_certificates_test/
Scylla version 0673e44fc1e6587e0154a7644f317a8c86e03fa4
```
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib64/python3.7/unittest/case.py", line 645, in run
    testMethod()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/native_transport_ssl_test.py", line 171, in reload_certificates_test
    session = self.patient_cql_connection(node1, ssl_opts={'ca_certs': os.path.join(self.test_path, 'ccm_node.cer'), "cert_reqs":ssl.CERT_REQUIRED})
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 866, in patient_cql_connection
    **kwargs
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 132, in retry_till_success
    return fun(*args, **kwargs)
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 772, in cql_connection
    **kwargs)
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 823, in _create_session
    session = cluster.connect()
  File "cassandra/cluster.py", line 1660, in cassandra.cluster.Cluster.connect
  File "cassandra/cluster.py", line 1696, in cassandra.cluster.Cluster.connect
  File "cassandra/cluster.py", line 1683, in cassandra.cluster.Cluster.connect
  File "cassandra/cluster.py", line 3470, in cassandra.cluster.ControlConnection.connect
  File "cassandra/cluster.py", line 3515, in cassandra.cluster.ControlConnection._reconnect_internal
cassandra.cluster.NoHostAvailable: ('Unable to connect to any servers', {'127.0.98.1:9042': PermissionError(1, "Tried connecting to [('127.0.98.1', 9042)]. Last error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1076)")})
```

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/490/artifact/logs-release.2/1590063995887_native_transport_ssl_test.NativeTransportSSL.reload_certificates_test/node1.log
```
INFO  2020-05-21 12:24:30,027 [shard 0] init - Scylla version 666.development-0.20200520.0673e44fc initialization completed.
INFO  2020-05-21 12:24:33,670 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
...
WARN  2020-05-21 12:25:33,761 [shard 0] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem}: std::_Nested_exception<std::runtime_error> (Could not read reloading /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem): std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: open failed: No such file or directory [/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem])
WARN  2020-05-21 12:25:33,761 [shard 0] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.key, /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem}: std::_Nested_exception<std::runtime_error> (Could not read reloading /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.key): std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: open failed: No such file or directory [/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.key])
Reactor stalled for 522 ms on shard 1.
Backtrace:
  0x0000000002e10b52
  0x0000000002dba084
  0x0000000002dba5b2
  0x0000000002dba8e7
  0x00007fc07ee9db1f
  0x00007fc07f583c0f
WARN  2020-05-21 12:25:34,499 [shard 1] cql_server - Exception loading {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem}: std::system_error (error GnuTLS:-34, Base64 decoding error.)
Reactor stalled for 522 ms on shard 1.
Backtrace:
  0x0000000002e10b52
  0x0000000002dba084
  0x0000000002dba5b2
  0x0000000002dba8e7
  0x00007fc07ee9db1f
  0x00007fc07f583bcf
Reactor stalled for 984 ms on shard 1.
Backtrace:
  0x0000000002e10b52
  0x0000000002dba084
  0x0000000002dba5b2
  0x0000000002dba8e7
  0x00007fc07ee9db1f
  0x00007fc07f583c1b
INFO  2020-05-21 12:25:35,633 [shard 1] cql_server - Reloaded {/jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.key, /jenkins/workspace/scylla-master/dtest-release/scylla/.dtest/dtest-di4gy39w/ccm_node.pem}
INFO  2020-05-21 12:25:35,656 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
INFO  2020-05-21 12:25:35,923 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
Reactor stalled for 522 ms on shard 1.
Backtrace:
  0x0000000002e10b52
  0x0000000002dba084
  0x0000000002dba5b2
  0x0000000002dba8e7
  0x00007fc07ee9db1f
  0x00007fc07f583bd6
INFO  2020-05-21 12:25:36,189 [shard 0] cql_server - exception while processing connection: std::system_error (error GnuTLS:-10, The specified session has been invalidated for some reason.)
...
```

**Comment by slivne** [comment]
@elcallio please check this out - we need some dtest coverage for this feature and its beeing disabled

**Comment by elcallio** [comment]
Caused/fixed by /scylladb/seastar#756

**Comment by bhalevy** [comment]
Verified with https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/175
